tests/unit/channel_tests.py

```diff
@@ -393,6 +393,18 @@ def test_basic_qos_raises_channel_closed(self):
         self.assertRaises(exceptions.ChannelClosed, self.obj.basic_qos, 0,
                           False, True)
 
+    def test_basic_qos_raises_prefetch_size_value_error(self):
+        self.assertRaises(ValueError, self.obj.basic_qos, None,
+                          'foo', 123, False)
+        self.assertRaises(ValueError, self.obj.basic_qos, None,
+                          -1, 123, False)
+
+    def test_basic_qos_raises_prefetch_count_value_error(self):
+        self.assertRaises(ValueError, self.obj.basic_qos, None,
+                          123, 'foo', False)
+        self.assertRaises(ValueError, self.obj.basic_qos, None,
+                          123, -1, False)
```

Comment:
In both of these tests, you need to validate that the parameters of the exception are what you expected not just the exception type. See example in https://github.com/pika/pika/blob/c0d0b3a2a69bdf394cd20d367a10a375e716e7b0/tests/unit/blocking_connection_tests.py#L149-L152. And you don't have to verify the verbatim exception text, but just some substring, such as `self.assertIn('prefetch_count', cm.exception.args[0])` (`ValueError` will have just the first string arg)